app/decorators/controllers/news/articles_controller_decorator.rb

```diff
@@ -0,0 +1,18 @@
+News::ArticlesController.class_eval do
+  before_action :require_login, only: [:destroy, :edit, :update, :new, :create, :admin]
+  before_action :authorization, only: [:destroy, :edit, :update, :new, :create, :admin]
+
+  layout 'layouts/application'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.